signac/contrib/migration/v1_to_v2.py

```diff
@@ -50,3 +50,13 @@ def _migrate_v1_to_v2(root_directory):
     v2_fn = _get_project_config_fn(root_directory)
     os.mkdir(os.path.dirname(v2_fn))
     os.rename(v1_fn, v2_fn)
+
+    # Now move all other files.
+    files_to_move = {
+        ".signac_shell_history": os.sep.join((".signac", "shell_history")),
+        ".signac_sp_cache.json.gz": os.sep.join((".signac", "sp_cache.json.gz")),
```

Comment:
If it's being renamed anyway, especially in a hidden folder, we can be more explicit about the name.
```suggestion
        ".signac_sp_cache.json.gz": os.sep.join((".signac", "statepoint_cache.json.gz")),
```